### DISCUSSION TITLE
redirect http:80 -> https

### DISCUSSION THREAD

**Discussion by Snipes999** [NginxProxyManager/nginx-proxy-manager#1592] [Q&A]
Hi,
I'm pretty new and started to test npm. In my old ngnix config I redirect every request on port 80 -> 443 with the following command:
return 301 https://$host$request_uri;

how could I achive this with npm to have all request on port 80 redirected to https port 443?
Thx & Regards,
Snipes

**Comment by chaptergy** [comment]
Enabling the "Force SSL" toggle under the SSL tab of the proxy host does exactly that.

**Reply by srvoets** [reply]
even with that toggled, mine isnt forcing ssl, I am using cloudflare with cloudflared
